index.html

```diff
@@ -5,12 +5,72 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="icon" href="Delicious-cupcake-PNG-1.png" type="image/icon type">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" rel="stylesheet" />
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css">
 </head>
 
 <body>
-    <!-- Add your markup here -->
+    <header>
+        <div class="logo">
+            <img src="Delicious-cupcake-PNG-1.png" alt="logo">
+            </div>
+            <nav>
+                <ul>
+                    <input src='checkbox' id='check' />
+                     <div class="menu">
+                <li><a href="#">Home</a></li>
+                <li><a href="#">Cakes</a></li>
+                <li><a href="#">Fantasy cakes</a></li>
+                <li><a href="#">Ordering</a></li>
+                <li><a href="#">About</a></li>
+                </div>
+                <label for="check" class="open-menu"><i class="fa-solid fa-bars"></i></label>
+            </ul>
+            </nav>
+            <p>Cupcakes & cakes Delivered to your door</p>
+    </header>
+    <main>
+        <section class="navigation">
+            <ul>
+                <li><a href="#">Home</a></li>
```

Comment:
Not a big problem, but something to consider - you could avoid repeating all the nav items twice in the HTML, by moving the nav panel with CSS (on small screens, hide it then hover below the burger menu when clicked; on larger screens, show it below the header).